### ISSUE TITLE
The name of NPC is UUID

### ISSUE THREAD

**Issue by CNSYGZS** [Foxikle/CustomNPCs#86]
When using customnpcs-1.6.1. jar on my server, the NPC name display includes my custom NPC name 1, my custom NPC name 2, and the NPC UUID display that I do not want NCP to display. I have tried using the NPC plugin's built-in name modification function, but I cannot modify or hide the NPC UUID display. I want to turn off the NPC UUID display on the NPC's head because it blocks the NPC's naming. My server does not have any other plugins installed for name modification, so I cannot manually turn off the NPC plugin's UUID display. I do not want the NPC UUID to be displayed, so I am seeking help here
![NPC](https://github.com/Foxikle/CustomNPCs/assets/165820698/2b6dc682-88a8-4d66-b2c2-b02d264f65df)

**Comment by Foxikle** [comment]
hey @CNSYGZS this is a known in compatibility with EssentailsX. Are you running EssentialsX? If you are you can temporarily disable the AFK feature while i work on the next update. If you are not running EssX can you run the `/team list` command to confirm the `npc` team exists? You might also want to try removing your client mods.

**Comment by CNSYGZS** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by CNSYGZS** [comment]
I don't think it's a problem with the EssentialsX plugin because I didn't have the EssentialsX plugin installed on my server simultaneously when using the NPC plugin. At least before, I didn't install both of them or the team plugin. In fact, I didn't have many or few plugins installed on my server. Currently, there is only one box store that is a plugin for expanding gameplay. The possibility of the problem being caused by the client UI beautification module is not high. I have used the basic client to log in to the server without installing the module. If you can solve this problem in the next version at the same time, we are happy. Perhaps you can separate the language configuration file in the next version together. I think I can provide you with Chinese and Chinese for free. This plugin is very popular in our country, Additionally, I have used a client without modules, light and shadow, and material packs to join my server. Please refer to the attached image when viewing NPCs
![无模组查看NPC](https://github.com/Foxikle/CustomNPCs/assets/165820698/70d52484-079a-4af8-8009-6e13f3d36b16)

**Comment by CNSYGZS** [comment]
In addition, I have tried using/team list NPC to view the server NPC team. This is a function of the server itself, not a plugin. The NPC names that exist in the attached image are all UUID. I used the/team remove NPC command to delete the NPC team and restart the server. However, when I tried again, the problem persisted
![teamlistnpc](https://github.com/Foxikle/CustomNPCs/assets/165820698/281f013e-bba1-4cd0-a9a2-b13833e33b95)

**Comment by CNSYGZS** [comment]
In addition, I have tried using/team list NPC to view the server NPC team. This is a function of the server itself, not a plugin. The NPC names that exist in the attached image are all UUID. I used the/team remove NPC command to delete the NPC team and restart the server. However, when I tried again, the problem persisted
![teamlistnpc](https://github.com/Foxikle/CustomNPCs/assets/165820698/da8ef147-19d3-4253-a648-ff0207454eb8)

**Comment by Foxikle** [comment]
I can definitely separate the messages into a translatable file. Can you try setting the nametag visibility to never? `/team modify npc nametagVisibility never` The only plugins on your server are CustomNPCs and a box store plugin?

Cheers!

**Comment by CNSYGZS** [comment]
> 我绝对可以将消息分离到一个可翻译的文件中。您可以尝试将名牌可见性设置为从不吗？ 您服务器上唯一的插件是 CustomNPC 和 box store 插件？`/team modify npc nametagVisibility never`
> 
> 干杯！

Yes, I have tried using the/team modify npc nametagVisibility never command to set the visibility of the name tag to never, but it prompted no change. The visibility of the name tag is already at this value [never]. After restarting the NPC plugin, I tried manually setting the visibility of the name tag to always display, and then tried again using the/team modify npc nametagVisibility never command to set the visibility of the name tag to never. I think something is preventing me from changing this value because when I try to adjust the visibility of the name tag, whether I change the never value to always value at the end of the command or when I try to set it to never again, its prompt remains unchanged. The visibility of the tag is already at this value, and I believe the NPC plugin will automatically adjust this value and its work in this part is correct, I will subscribe to your GitHub. If you later separate the language file from the configuration file in the customnpcs plugin, I will translate it into Chinese as soon as I see it, and then submit the language configuration after Chinese translation to you in GitHub. You can check it and add the translated GUI and other Chinese configuration files to the customnpcs plugin at the appropriate time (you think). I am happy about it for free because we all like the plugin you have created

**Comment by CNSYGZS** [comment]
Can you tell me if I can modify the corresponding NPC's UUID to a different name or hide comments in the npcs.yml file of the configuration file I am submitting for the customnpcs plugin I am currently using? I am not sure if this will result in errors or any catastrophic errors. My plugin list is BedrockArmorstands-1.2.5.jar BackLoc.jar customnpcs-1.61.jar DecentHolograms-2.8.8.jar LuckPerms Bukkit 5.4.128.jar QuickShop Hikari-62.0.3.jar Default. jar PlaceholderAPI 2.11.6.jar ProtocolLib.jar Warp.jar
[CustomNPCs.zip](https://github.com/user-attachments/files/16068195/CustomNPCs.zip)

**Comment by Foxikle** [comment]
Absolutely DO NOT modify the uuid. That would indeed be catastrophic. I'm going to look into all of those plugins, because I suspect one of them is messing with the teams, or how I hide the nametags).

**Comment by CNSYGZS** [comment]
> 绝对不要修改 uuid。这确实是灾难性的。我将研究所有这些插件，因为我怀疑其中一个插件正在扰乱团队，或者我如何隐藏名牌）。

Hi, it has been some time and I have tested using different clients. Only the client using Geyser cannot see the NPC's UUID. If you find any new solutions, please let me know

**Comment by Foxikle** [comment]
Hi! I did find a solution, but some other parts of the plugin, mostly the menus are really unstable. I also began working on implementing translation keys.

Sorry for the delay.

**Comment by CNSYGZS** [comment]
> 你好！我确实找到了解决方案，但是插件的其他一些部分，主要是菜单真的很不稳定。我还开始致力于实现翻译键。
> 
> 很抱歉耽搁了。

I am very pleased that you have resolved this issue. I will make a copy of the language configuration file that you separated in your next update, translate it into Chinese, and submit it as soon as possible. Solving this issue will indeed take some time, and I thought you forgot about it. Now it seems that you have not forgotten and have started the repair work. I hope my email did not disturb you. I am just curious about the progress of the repair. Thank you very much.

**Comment by FluffyCory** [comment]
Hello, I have the same error, but I'm not sure if I understand the connection correctly and I definitely didn't understand how you solved the problem.
could you help me please?

**Comment by CNSYGZS** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by Foxikle** [comment]
I fixed the issue by mounting another text display as a passenger of the player entity. Its not really ready for release yet, as I'm doing a bunch of rewrites to some internals.

**Comment by FluffyCory** [comment]
Good, so it's enough to wait for the update :D hopefully it will work.

**Comment by FluffyCory** [comment]
I already figured out which plugin displayed the uuid. It was a plugin called Xg7Lobby.
At least that's how it is for me

**Comment by Foxikle** [comment]
If that plugin uses teams, that would explain it. I am still waiting on the developer to fix a few big bugs then I can release it.

**Comment by Foxikle** [comment]
Hey! Sorry for the long wait, but I finally fixed the issue!
https://github.com/Foxikle/CustomNPCs/releases/tag/v1.7-pre3

If you are still interested in translating, feel free to check out the project on [Crowdin](https://crowdin.com/project/customnpcs).

Thanks for sticking through :)

**Comment by CNSYGZS** [comment]
Okay, I saw the latest content you posted. I didn't check my email during my vacation. Now I'm starting to localize the language configuration file. After checking, I will submit the new language configuration file to Crowdin

**Comment by shonidan** [comment]
The bug persist when I put the command /npc reload, but disappear after a 3 minutes approx.

**Comment by CNSYGZS** [comment]
Are you sure you have updated to the latest version?

**Comment by CNSYGZS** [comment]
I have submitted the latest translation file

**Comment by Foxikle** [comment]
> I have submitted the latest translation file

Great! I'll take a look when ai get a chance. Thank you for translating! If you are already in my discord server, or would like to join it, I will give you access to the channel where I post the preview builds. 

Cheers!

**Comment by shonidan** [comment]
Hello again, first of all thanks for this fantastic plugin, I have the latest version, the following happens to me, I have 2 NPCs with the same skin and I notice that their name disappears, when I select /npc reload, the names appear but behind them you can see the UUID.

**Comment by Foxikle** [comment]
Can you create a new issue? Please include a screenshot of /npc and /plugins
